### ISSUE TITLE
multiline inherit will return false positives

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#121]
```
inherit autotools \
        gobject-introspection \
        pkgconfig \
```
will complain about `\x1b` being inherited multiple times - which is non-sense.

Reference https://github.com/priv-kweihmann/meta-sca/issues/1031
